Description: commonjs plugin has to be applied outside node_modules
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2022-04-25

--- a/rollup.config.ts
+++ b/rollup.config.ts
@@ -45,7 +45,6 @@
 	string({ include: '**/*.md' }),
 	commonjs({
 		ignoreTryCatch: false,
-		include: 'node_modules/**'
 	}),
 	typescript(),
 	cleanBeforeWrite('dist')
